windows/cuda116.bat

```diff
@@ -16,7 +16,7 @@ IF ERRORLEVEL 1 goto :eof
 REM Check for optional components
 
 set USE_CUDA=
-set CMAKE_GENERATOR=Visual Studio 15 2017 Win64
+set CMAKE_GENERATOR=Visual Studio 17 2022 Win64
```

Comment:
I've updated only CUDA 11.6 and 11.7 as those are listed in the build selection table on the homepage (https://pytorch.org/)